client/src/utils.ts

```diff
@@ -139,3 +141,81 @@ export function onVisibilityChange(): void {
     });
   }
 }
+
+function getTopicsParams(url = location.href) {
+  // Create a params object
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const recommendedTopics: any = {};
```

Comment:
can't the type for this object be something like:
`Record<string, string | string[]>`

Reply:
That was my first implementation but for some reason, it doesn't allow to use of that `Record` type, so that's why added any.

Reply:
let me give it a shot

Reply:
[d1a5fe4](https://github.com/mentorpal/mentor-client/pull/171/commits/d1a5fe49ac8c763faf276a7335e5b2f5784897ad)